src/components/ToggleButton/ToggleButton.js

```diff
@@ -82,15 +100,7 @@ export default class ToggleButton extends Component {
     }
 
     if (variant === 'grid-list') {
```

Comment:
Also, sorry I didn't see this at first, but I'm thinking we should just have a switch statement for variant at this point, where `default:` handles all the non-special-case variants.